### DISCUSSION TITLE
Right border length

### DISCUSSION THREAD

**Discussion by pradyuk** [twilio-labs/paste#799] [Q&A] [answered]
I'm trying to make my right border touch the top and bottom of my blue background, but a bit stuck with that. Also, when I click, the focus state is not wrapping around the icon only, it stretches to the right border which is not desired. Any help will be appreciated.
![Screen Shot 2020-09-25 at 11 35 12 AM](https://user-images.githubusercontent.com/11803735/94304108-cd359300-ff23-11ea-9858-ebfdf945acb5.png)
![Screen Shot 2020-09-25 at 11 36 04 AM](https://user-images.githubusercontent.com/11803735/94304110-ce66c000-ff23-11ea-8390-f9ae3b818711.png)

Link to PR -> [click here](https://code.hq.twilio.com/twilio/console/pull/91)

**Comment by serifluous** [comment]
Thanks, @pradyuk ! Can you push your changes so we can see how you've implemented the border?

**Reply by pradyuk** [reply]
My bad, pushed! :)

**Comment by serifluous** [comment]
I'll check in with our engineering team on this border question, but a quick unrelated thing: You might already have this link but this has the tokens you'll need for things like the background color and the border color spec'd out https://share.goabstract.com/b72d29bc-9dbf-4f99-9a52-d3b4e6bf53cd?collectionLayerId=efe4c114-c1ec-47a1-a35d-8d2f2ad50c78&mode=design

(ignore me if you just haven't gotten around to this level of styling yet)

**Reply by pradyuk** [reply]
Thanks for that @serifluous! I had that link, but just started using abstract so figured about all that info after your msg! I'll still need a little help with that border though :)

**Comment by richbachman** [comment, marked as answer]
Hi @pradyuk it looks like your `Box` may be in the place. I put together a quick CodeSandbox to show a possible solution. https://codesandbox.io/s/header-box-example-lo35j?file=/src/index.tsx

**Reply by pradyuk** [reply]
Thanks again @richbachman! You're right, the box was in place and the theme helped as well. The focus state is going off again though. I tried using the display="block", but no luck.

**Reply by richbachman** [reply]
Right now theres a bug in Anchor that is blocking the `display="block"`. That should be fixed very soon. Once it's released you'll need to update the version of core. I can ping you when that release goes out.

**Reply by pradyuk** [reply]
That will be great, thanks again! 👍

**Reply by richbachman** [reply]
Hi @pradyuk we just merged the fix for the display prop on Anchor, so if you update your packages you should be good to go.

**Reply by pradyuk** [reply]
Hi @richbachman! Thanks for letting me know. I tried updating the packages, and iconColor is failing for some reason. "Property 'iconColor' does not exist on type 'IntrinsicAttributes & IconWrapperProps & { icon: string; title?: string | undefined; decorative: boolean; }"

**Reply by serifluous** [reply]
Try `color` rather than iconColor. The icon component props table might be able to help you out: https://paste.twilio.design/icons/usage-guidelines#props

**Reply by pradyuk** [reply]
Looks like there is a bug. I was actually using iconColor before and it was working fine. When I try now, I get the desired result on storybook with iconColor but i see an error in the editor. Using "color" is not giving be the desired output. Seems a bit weird.

**Reply by richbachman** [reply]
Hi @pradyuk I think we might be seeing a dependency issue now. You'll need to update all of the Paste packages in order to get the correct types. The full list is here: https://paste.twilio.design/getting-started/engineering#install-paste-system-packages

Keeping these up to date will be much easer soon.

**Reply by pradyuk** [reply]
@richbachman yup, that was the issue. Thanks! :)
